models.js

```diff
@@ -48,4 +48,30 @@ function fetchArticles() {
     });
 }
 
-module.exports = { fetchTopics, fetchArticleById, fetchArticles };
+function fetchCommentsByArticleId(article_id) {
+  return db
+    .query(
+      `SELECT
+        comment_id, votes, created_at, author, body, article_id
+    FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC;`,
+      [article_id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: "Comments not found for this article",
+        });
+      }
```

Comment:
If you are adding the new test for an existing article that has _no comments_ this logic will need to be revised. Currently this logic will respond with an error whenever there are no comments, but if the article does exist we would hope for an empty array `[]` and `200`. 
_note: this may require a separate query to check if the article can be found before fetching the comments_